workers/discourse_analysis_worker/discourse_analysis_worker.py

```diff
@@ -16,7 +15,6 @@
 from sklearn_crfsuite import scorers
 from sklearn_crfsuite import metrics
 from sklearn.metrics import make_scorer
-from sklearn.model_selection import cross_val_score
```

Comment:
In this worker we are importing specific components of the libraries.

Reply:
ok will reverse the change